src/resourcesManager.ts

```
@@ -38,13 +37,23 @@ export class LoadedResources {
 
-  itemsRenderer: ItemsRenderer
-  worldBlockProvider: WorldBlockProvider
+  itemsRenderer: ItemsRenderer | undefined
+  worldBlockProvider?: WorldBlockProvider
   blockstatesModels: any = null
 
   version: string
   texturesVersion: string
+
+  prepareForTransfer () {
+    delete this.itemsRenderer
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)